yarn.lock

```diff
@@ -5,7 +5,6 @@
 "@babel/cli@^7.5.5":
   version "7.5.5"
   resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.5.5.tgz#bdb6d9169e93e241a08f5f7b0265195bf38ef5ec"
-  integrity sha512-UHI+7pHv/tk9g6WXQKYz+kmXTI77YtuY3vqC59KIqcoWEjsJJSG6rAxKaLsgj3LDyadsPrCB929gVOKM6Hui0w==
```

Comment:
Why are all the integrity strings removed? Are you using an old version of Yarn?

Reply:
Thanks for spotting. Something weird had been going on on my local machine. I have been updating yarn, but it was actually failing to switch to the newer version. I've now managed to do so, and produced a new `yarn.lock` file